frontend/packages/dev-console/src/test/pipeline-data.ts

```diff
@@ -786,3 +787,134 @@ export const pipelineTestData: PipelineTestData = {
     },
   },
 };
+
+export const mockPipelineRun = {
```

Comment:
Would have preferred you put it in the `pipelineTestData` map or reused one of the ones there... but at least it's in the right file.

Reply:
Reused a pipelinerun from the map.